nixos/modules/services/hardware/asusctl.nix

```diff
@@ -5,10 +5,10 @@ with lib;
 let
   cfg = config.services.asusctl;
 
-  animeConfigFile = pkgs.writeText "anime.conf" (if cfg.animeConfig == { } then "" else builtins.toJSON cfg.animeConfig);
-  asusdConfigFile = pkgs.writeText "asusd.conf" (if cfg.asusdConfig == { } then "" else builtins.toJSON cfg.asusdConfig);
-  auraConfigFile = pkgs.writeText "aura.conf" (if cfg.auraConfig == { } then "" else builtins.toJSON cfg.auraConfig);
-  profileConfigFile = pkgs.writeText "profile.conf" (if cfg.profileConfig == "" then "" else builtins.toJSON cfg.profileConfig);
+  animeConfigFile = pkgs.writeText "anime.conf" (lib.optionalString (cfg.animeConfig != { }) builtins.toJSON cfg.animeConfig);
```

Comment:
This breaks eval now. Should be `(lib.optionalString (condition) (builtins.toJSON cfg.animeConfig))`